code/modules/mob/living/simple_animal/friendly/hamster.dm

```diff
@@ -45,7 +45,7 @@
 	if(prob(5))
 		var/datum/disease/disease = pick(/datum/disease/cold, /datum/disease/flu, /datum/disease/fluspanish)
 		vector_disease = new disease
-		message_admins("Vector was roundstart infected with [vector_disease.name]. Don't lynch the virologist!")
```

Comment:
I say soul removed because I see this message all the time in local testing since it happens during init and you start adminned on local